styles/styles.less

```diff
@@ -1,5 +1,9 @@
 @import "variables";
 
+atom-pane-container atom-pane div.item-views {
```

Comment:
Hmm I couldn't find a way to get that work. I'll just remove this line and leave it as is for now